encrypted-media-respec.html

```diff
@@ -694,15 +694,23 @@
                   </dd>
                   <dt><a def-id="requirement-optional"></a></dt>
                   <dd>
-                    <p>Continue.<!-- Will be updated below. --></p>
+                    <p>If <a def-id="distinctive-identifiers"></a> are not allowed according to <var>restrictions</var>:</p>
+                    <ol>
+                      <li>
+                        <p>Set the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> to <a def-id="requirement-not-allowed"></a></p>
+                      </li>
+                      <li>
+                        <p>If the <var>implementation</var> requires a <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var> and <var>restrictions</var>, return <code>NotSupported</code>.</p>
+                      </li>
+                    </ol>
                   </dd>
                   <dt><a def-id="requirement-not-allowed"></a></dt>
                   <dd>
                     <p>If the <var>implementation</var> requires a <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var> and <var>restrictions</var>, return <code>NotSupported</code>.</p>
                   </dd>
                 </dl>
               </li>
-              <li><p>Set the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> to equal <var>candidate configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> member.
+              <li><p>If the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> has not been set, set it to equal <var>candidate configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> member.
```

Comment:
Alternatively, we could insert a step to set a local variable to the "`the value of <var>candidate configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> member.`" then just set accumulated configuration's value to that.

Although it adds a step, I think it's simpler than setting accumulated configuration's value in two places.